topcoffea/modules/event_selection.py

```diff
@@ -26,11 +26,13 @@ def passes_trg_inlst(events,trg_name_lst):
 #   - Returns an array the len of events
 #   - Elements are false if they do not pass any of the triggers defined in dataset_dict
 #   - In the case of data, events are also false if they overlap with another dataset
-def trg_pass_no_overlap(events,is_data,dataset,year,dataset_dict,exclude_dict):
+def trg_pass_no_overlap(events,is_data,dataset,year,dataset_dict,exclude_dict,era=None):
 
-    # The trigger for 2016 and 2016APV are the same
+    # The triggers for 2016 and 2016APV are the same
     if year == "2016APV":
         year = "2016"
+    if year == "2022EE":
```

Comment:
But the trigger lists are not the same for 2022 and 2022 EE, right? So I am not sure if I understand the impact this will have. Could you explain a bit about what's happening here?

Reply:
I think we should leave this: the trigger list is the same, the problem with 2022 & 2022EE is the sudden change from SIngleMuon and DoubleMuon to just Muon. However, Muon still consists of the triggers from SingleMuon and DoubleMuon. And the triggers for EGamma and MuonEG are all the same

Reply:
Ok, thanks for explaining. For the record, now that I look at this function again, I think it would be better if we handed this year stuff on the users' end where we _call_ this function (e.g. just pass "2022" for the year in the case where the year is "2022EE" instead of having this hard coded in the function itself. 
But I'd say not necessary to fix in this PR. I've made an [issue](https://github.com/TopEFT/topcoffea/issues/24) so that we remember it in the future (if we'd like to address it someday).